ArjunaJTA/object_store/src/main/java/org/jboss/narayana/jta/quickstarts/VolatileStoreExample.java

```diff
@@ -45,8 +46,7 @@ public static void main(String[] args) throws Exception {
     public static void setupStore() throws Exception {
         defaultStoreDir = BeanPopulator.getDefaultInstance(ObjectStoreEnvironmentBean.class).getObjectStoreDir();
 
-        BeanPopulator.getNamedInstance(ObjectStoreEnvironmentBean.class, "default").setObjectStoreType(storeClassName);
-        BeanPopulator.getNamedInstance(ObjectStoreEnvironmentBean.class, "communicationStore").setObjectStoreType(storeClassName);
+        new MetaObjectStoreEnvironmentBean().setObjectStoreType(storeClassName);
```

Comment:
MetaObjectStoreEnvironmentBean was added to ensure that our config bean changes were backwardly compatible with the old arjPropertyManager approach to config (ie did not break the old semantics of arjPropertyManager).

I would either keep the old code or use arjPropertyManager instead (but the problem then is that we want to discourage the use of arjPropertyManager)

Reply:
I see. I needed to make an addition to code to set the type for the default instance. Otherwise I've left the code as it was.

Now I have a question how is the recommended way to access the configuration bean. I usually use the access through the `arjPropertyManager` and I haven't realized that the `BeanPopulator` should be used instead.
If the `BeanPopulator` is the prefered way for setting-up the properties then maybe all the quickstarts should be changed to use it instead of `arjPropertyManager`, `recoveryPropertyManager` etc. Or the both are equal and I should not care?

Reply:
I see, thanks for clarification. Totally agree with raising different JIRA/PR for that purpose (but I do not plan to do so now).

Reply:
If you are reading or writing properties of none default config beans programatically then the only option is to use the BeanPopulator. On the other hand we document how to do it via config files:

[Section 3.5 of the guide](http://narayana.io//docs/product/index.html#d0e3473) talks about setting properties via the EnvironmentBean classes and then states that setting properties using the form 

> NameEnvironmentBean.propertyName (this is the preferred form used in the properties file)

is preferred. It also mentions how to read/write properties of none default beans: 

> In specific cases when you want to set properties on configuration beans other that the default > bean instances the form is EnvironmentBeanClass.<storeType>.propertyName.